types/src/block_executor/config.rs

```diff
@@ -4,6 +4,31 @@
 use crate::on_chain_config::BlockGasLimitType;
 use serde::{Deserialize, Serialize};
 
+/// Local, per-node configurations for module cache. While caches can be persisted across multiple
+/// block executions, these configurations allow to specify cache sizes, etc.
+#[derive(Clone, Debug)]
+pub struct BlockExecutorModuleCacheLocalConfig {
+    /// If true, when global caches are empty, Aptos framework is prefetched into module cache.
+    pub prefetch_framework_code: bool,
+    /// The maximum size of module cache (the sum of serialized sizes of all cached modules in
+    /// bytes).
+    pub max_module_cache_size_in_bytes: usize,
+    /// The maximum size (in terms of entries) of struct name re-indexing map stored in the runtime
+    /// environment.
+    pub max_struct_name_index_map_num_entries: usize,
```

Comment:
can we add counters (i.e. IntGauge) to be populated after every block execution is done, to export module_cache_size_in_bytes and struct_name_index_map_num_entries ?

Reply:
Added counters